interface/src/app/gene/searchMany/searchMany.js

```diff
@@ -25,6 +25,15 @@ angular.module('adage.gene.searchMany', [
               // be obtained from the ML model. This is the same as
               // the issue in geneSearchForm (also with $scope.organism).
               $scope.organism = 'Pseudomonas aeruginosa';
+
+              // '$scope.autocomplete.autocomlete' holds a boolean value,
+              // of whether or not the user wants to use autocomplete
+              // search to look up a few genes. If not, then the autocomplete
+              // search panel will disappear, and the panel to search many
+              // genes will appear. We need to send the value itself inside
+              // an object so that it gets sent properly back and forth
+              // through the corresponding $scopes.
+              $scope.autocomplete = {autocomplete: true};
```

Comment:
@dongbohu: Yes, I agree - In fact, the code is already using the controllerAs syntax, I changed it to using that syntax last week because I also thought that that was the best way to go. I think GitHub is just not showing those updates in this specific view, but they will show if you look at the Files changed. Does this look good to you as it is now?